independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/jbang/JBangDevModeLauncherImpl.java

```diff
@@ -174,4 +179,19 @@ private static String getQuarkusVersion() {
             throw new RuntimeException(e);
         }
     }
+
+    private static Properties getConfigurationProperties(final Path source) throws IOException {
+        Properties properties = new Properties();
+        for (String line : Files.readAllLines(source)) {
+            if (line.startsWith("//Q:CONFIG")) {
+                String conf = line.substring(10).trim();
+                int equals = conf.indexOf("=");
+                if (equals == -1) {
+                    throw new RuntimeException("invalid config  " + line);
+                }
+                properties.setProperty(conf.substring(0, equals), conf.substring(equals + 1));
```

Comment:
IIUC when this is set, there is no point to continue, so maybe `break` the loop.